### PR TITLE
Doc: Move 32-bit Linux from top of suggested builds

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: doc/charm++/manual.rst [hunk 1]
```diff
@@ -3852,7 +3852,7 @@ manner:
 .. code-block:: c++
 
      msg->arr1[13] = 1;
-     msg->arr2[5] = 32.82;
```
Comment (matthiasdiener) [inline doc/charm++/manual.rst:3855]: Is this change intentional?

Comment (rbuch) [reply doc/charm++/manual.rst:3855]: Nope, I'll revert it. (I did this in GitHub's editor and I must have messed up when searching for "32".)
